.github/ISSUE_TEMPLATE/7_cherry_pick.md

```diff
@@ -0,0 +1,24 @@
+---
+name: Request a cherry-pick.
+about: As a contributor, you would like to request that a feature be cherry-picked into a release.
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+commit(s) to merge: (Enter the shortened commit hash of the commit that has been merged to master/main. <fabfeed>)
+
+merge instructions: clean merge
+
+What is the issue: (Enter brief description of the issue)
+
+What is the fix: (What does this cherry-pick fix?)
+
+Why cherrypick: (Why should this be cherry-picked and not exist as part of a regular release?)
+
+Risk: (<Low, Medium, High>, Has this been merged into flutter/flutter:master, flutter/engine:main etc?)
+
+Link to original issue(s): (What is the issue number that this cherry-pick request solves? ex. flutter/flutter#12345)
+
```

Comment:
May also be good to have an optional "Steps to manually validate fix" if possible to manually test on the hotfix branch (it's not always possible).  It's still amazing to me that we don't do that--we just cherry-pick it and if the tests pass usually no one does a manual smoke test to see if the general product still works, or if the bug is fixed.